tests/drizzle/test_drizzle.py

```diff
@@ -1,52 +1,211 @@
 import pytest
-
 import numpy as np
 from astropy import wcs
-
 from drizzlepac import cdriz
 
-@pytest.mark.parametrize(
-    'kernel', ['square', 'point', 'turbo', 'gaussian', 'lanczos3']
-)
-def test_zero_input_weight(kernel):
+
+# testing set up
+class Get_Grid:
+    def __init__(self):
+        def get_wcs(_grid):
+            w = wcs.WCS()
+            w.wcs.ctype = ["RA---TAN", "DEC--TAN"]
+            w.wcs.crpix = [_grid[0], _grid[1]]
+            w.wcs.crval = [10, 10]
```

Comment:
During our discussion with Warren, did he suggest the CRVAL values be larger (i.e., ˜100)?

Reply:
I'm pretty sure that he was suggesting that the pixel being smoothed be brighter, so setting the selected insci pixel to 100 as opposed to 10. That being said, I don't really have a sense for how changing the crval changes the results.

Reply:
The CRVAL position specifies the RA/Dec value of the reference pixel in the image, where the reference pixel is the pixel in the input image where the distortion is zero.  The ref pixel is usually the center of the image.  In addition, I had originally suggested increasing the flux in the input pixel values from 10 to 100 or 1000 or even 10000 simply to make any splitting of the pixel's flux easier to recognize in the output frame.  This was only a suggestion, not a correction of any problem with the original test.

Reply:
Add CRVALS as decimals (ra/dec in decimal degrees)